core/command/report.js

```diff
@@ -77,6 +77,29 @@ function writeBrowserReport (config, reporter) {
   logger.log('Writing browser report');
 
   return fs.copy(config.comparePath, toAbsolute(config.html_report)).then(function () {
+    // Slurp in logs
```

Comment:
Should this block run if user opts out of log writing?

Reply:
I ran into some problems in testing where the config changes between the reference and test runs, meaning that if the config is set _now_, it doesn't tell us if it was set _then_. So the log manipulating code in this block catches and ignores errors while reading the log files.

We'll need that anyway for the case where there aren't log files as part of the reference, but there are as part of the test. We could skip the block if the config is currently disabled; I'll make that change.